homework-submission/todo-app/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { inject, observer } from 'mobx-react';
+import Form from './Components/Form';
+import List from './Components/List';
+import './App.css';
+
+@inject('TasksStore')
+@observer
```

Comment:
I would prefer to inject in the Form component as a one way code style.

because you are injecting here and in the `List` component, while you don't have a use of the store actions you are passing, but instead you are injecting. 💔